stable/cni-genie/templates/daemonSet.yaml

```diff
@@ -0,0 +1,156 @@
+# Install CNI-Genie plugin on each slave node.
+kind: DaemonSet
+apiVersion: extensions/v1beta1
```

Comment:
why are you using an older api version? does this workload not supported in the latest?  for charts it is recommended to start with Latest version -1 or Latest version - 2